google/cloud/internal/oauth2_external_account_credentials.cc

```diff
@@ -96,10 +96,19 @@ StatusOr<ExternalAccountInfo> ParseExternalAccountConfiguration(
       MakeExternalAccountTokenSource(*credential_source, *audience, ec);
   if (!source) return std::move(source).status();
 
-  auto info =
-      ExternalAccountInfo{*std::move(audience),  *std::move(subject_token_type),
-                          *std::move(token_url), *std::move(source),
-                          absl::nullopt,         *std::move(universe_domain)};
+  absl::optional<std::string> workforce_pool_user_project;
+  auto up_it = json.find("workforce_pool_user_project");
+  if (up_it != json.end()) {
+    workforce_pool_user_project = *up_it;
```

Comment:
nit: I think this is doing some implicit conversion from `nlohmann::json` -> `std::string`.

It seems like we prefer to do something explicit to access string fields: `up_it->get<std::string>()`

Reply:
Done.